src/components/News.js

```diff
@@ -0,0 +1,48 @@
+import { Fragment } from "react";
+import { useParams } from "react-router";
+
+
+export default function News({
+    newsImg,
+    newsTitle,
+    newsDate,
+    newsTekst,
+}){
+
+    const { newsId: newsId } = useParams();
+
+    const getNewsData = async () => {
```

Comment:
dead code?

Reply:
We might end up need it later on, if we end up having a page for news

Reply:
@mircealungu this is an example of a comment.